hoomd/md/constrain.py

```diff
@@ -4,192 +4,49 @@
 # Maintainer: joaander / All Developers are free to add commands for new
 # features
 
-R""" Constraints.
+r""" Constraints.
```

Comment:
```suggestion
r"""Constraints.
```
Normally flake8-docstrings is picky about this.